src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-width;
+  width: calc($width-day * 7 + 10px * 2 + $gap-width * 6);
+  padding: 10px;
+
+  &__day {
+    position: relative;
+    width: $width-day;
+    height: $height-day;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        font-size: 30px;
+        color: black;
+        font-family: Arial, Helvetica, sans-serif;
+        top: 50%;
+        left: 50%;
+        transform: translate(-49%, -50%);
```

Comment:
The transform property in the ::before pseudo-element uses a hard-coded value to center the content. Consider using `transform: translate(-50%, -50%);` to perfectly center the content regardless of the font size or content width.